lib/request-logger.js

```diff
@@ -12,8 +11,27 @@ exports.register = function (server, options, next) {
   })
 
   server.ext('onPreResponse', (request, reply) => {
-    const latency = Date.now() - request.timing.start
-    request.logger.info(toCommonLogFormat(request, options), `${latency}ms`)
+    const now = Date.now()
+    const rawRequest = request.raw.req
+
+    let clientIp = request.info.remoteAddress
+    if (options && options.ipHeader && request.headers[options.ipHeader]) {
+      clientIp = request.headers[options.ipHeader]
+    }
+
+    const latency = now - request.timing.start
+    const method = rawRequest.method
+    const path = rawRequest.url
+    const protocol = rawRequest.client
+      ? rawRequest.client.npnProtocol
+      : 'HTTP/' + rawRequest.httpVersion
+    const statusCode = request.response.statusCode || request.response.output.statusCode
+    const userAgent = request.headers['user-agent']
```

Comment:
I added the user agent as well. It bothered me that Common Log Format didn't capture this.